src/main/frontend/components/page_menu.cljs

```diff
@@ -78,7 +78,8 @@
           file-sync-graph-uuid (and (user-handler/logged-in?)
                                     (file-sync-handler/enable-sync?)
                                     (file-sync-handler/get-current-graph-uuid))]
-      (when (and page (not block?))
+      (when (and page (not (or (block?)
+                               (config/publishing?))))
```

Comment:
`config/publishing?` is a boolean so no need to wrap in parenthesis. Same for `block?` above. Fwiw the lint job caught this